--- src/yubico-piv-tool-2.6.1/lib/internal.c.orig	2024-09-12 05:51:31.000000000 -0500
+++ src/yubico-piv-tool-2.6.1/lib/internal.c	2024-11-06 16:23:00.445771700 -0600
@@ -33,6 +33,7 @@
 #include <windows.h>
 #include <bcrypt.h>
 #include <strsafe.h>
+#include <openssl/evp.h>
 #else
 #include <ctype.h>
 #include <syslog.h>
@@ -364,7 +365,7 @@
 pkcs5_rc pkcs5_pbkdf2_sha1(const uint8_t* password, const size_t cb_password, const uint8_t* salt, const size_t cb_salt, uint64_t iterations, const uint8_t* key, const size_t cb_key) {
   pkcs5_rc rc = PKCS5_OK;
 
-#ifdef _WIN32
+#if 0
   /* mingw64 defines the BCryptDeriveKeyPBKDF2 function, but its dll link library doesn't include the export.
   **
   ** In case this is needed, we'll need to dynamically load the function:
