--- src/yubico-piv-tool-2.6.1/ykcs11/CMakeLists.txt.orig	2024-09-12 05:51:31.000000000 -0500
+++ src/yubico-piv-tool-2.6.1/ykcs11/CMakeLists.txt	2024-11-06 16:52:57.379297100 -0600
@@ -78,11 +78,7 @@
 if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     set_target_properties(ykcs11_shared PROPERTIES INSTALL_RPATH "${YKPIV_INSTALL_LIB_DIR}")
 endif()
-if(WIN32)
-    set_target_properties(ykcs11_shared PROPERTIES OUTPUT_NAME libykcs11)
-else(WIN32)
-    set_target_properties(ykcs11_shared PROPERTIES OUTPUT_NAME ykcs11)
-endif(WIN32)
+set_target_properties(ykcs11_shared PROPERTIES OUTPUT_NAME ykcs11)
 add_coverage(ykcs11_shared)
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/ykcs11.pc.in ${CMAKE_CURRENT_SOURCE_DIR}/ykcs11.pc @ONLY)
